.circleci/config.yml

```diff
@@ -328,6 +345,16 @@ jobs:
             docker tag 657871693752.dkr.ecr.us-east-1.amazonaws.com/filecoin-faucet:$SHORT_GIT_SHA 657871693752.dkr.ecr.us-east-1.amazonaws.com/filecoin-faucet:latest
             docker push 657871693752.dkr.ecr.us-east-1.amazonaws.com/filecoin-faucet:latest
 
+      - run:
+          name: build & push image - filecoin
+          command: |
+            mv $HOME/filecoin-proof-parameters ./filecoin-proof-parameters
```

Comment:
How are the params being generated at `$HOME/filecoin-proof-parameters`. I don't see anything that would change them from their default of `/tmp/filecoin-proof-parameters`. You can use `FILECOIN_PARAMETER_CACHE` to change it though. I believe setting it during `deps` should work.

Reply:
@travisperson it gets set here https://github.com/filecoin-project/go-filecoin/blob/master/.circleci/config.yml#L178

I had found that out the hard way when trying to symlink the groth parameters in `smartdeps` and they weren't found in `/tmp` in CI

Reply:
Echoing @travisperson's observations, while the files in `filecoin-proof-parameters/` will be present in the `docker build` context, I believe that we also need to set `FILECOIN_PARAMETER_CACHE=filecoin-proof-parameters` in  `Dockerfile` or as a `build-arg` so that the build process sees that those files already exist and don't need to be regenerated

Reply:
Ah it was already set prior to this PR see line 15

```
echo 'export FILECOIN_PARAMETER_CACHE="${HOME}/filecoin-proof-parameters/"' >> $BASH_ENV
```

Reply:
that only sets the ENV for the current shell running on CI, it doesn't automatically get set inside the Docker build AFAIK

Reply:
Sorry, I jumped over your comment, I was replying to @ognots and my original.

Reply:
@gmasgras @travisperson the proof parameters get pulled in via docker `COPY` https://github.com/filecoin-project/go-filecoin/blob/f6a77f7a22199f84b0d2c62ad3b436ebc6ed553d/Dockerfile.ci.filecoin#L6
and the parameters are moved to docker context here https://github.com/filecoin-project/go-filecoin/blob/f6a77f7a22199f84b0d2c62ad3b436ebc6ed553d/.circleci/config.yml#L373

Reply:
the files are moved to `/tmp/filecoin-proof-parameters` as expected by default.

it might be better to copy the proof params to `/usr/share/filecoin-proof-parameters` and then set `ENV FILECOIN_PARAMETER_CACHE /usr/share/filecoin-proof-parameters` in Dockerfile